packages/frontend/components/Plan/AddPlanModal.tsx

```diff
@@ -296,6 +306,20 @@ export const AddPlanModal: React.FC<AddPlanModalProps> = ({
                         }}
                       />
                     )}
+                    {majorName && !isValidatedMajor && (
```

Comment:
If a major name is not selected, the checkbox will show up by default because isValidatedMajor will default to false. I assume we only want to show the checkbox if the selected major is a beta major.

Reply:
Can we leave a quick comment about this above this line as i don't think its that obvious